fog/ledger/server/tests/router_integration.rs

```diff
@@ -166,22 +165,9 @@ fn create_store(
     store
 }
 
-fn create_shard(config: &ShardConfig, _logger: Logger) -> ShardProxyServer {
```

Comment:
A bit of overloaded terminology the "shards" in this test were trying to mimic  "load balancers" with multiple store copies. Unfortunately they mimicry failed to represent real world usage as the routers won't balance, they latch on to which ever store copy they first talk to.